tsup.config.ts

```diff
@@ -0,0 +1,10 @@
+import { defineConfig } from "tsup";
+
+export default defineConfig({
+	bundle: false,
+	clean: true,
+	dts: true,
+	entry: ["src/**/*.ts", "!src/**/*.test.*"],
+	format: ["cjs", "esm"],
+	outDir: "dist",
```

Comment:
Both `dist` and `lib` are totally reasonable outputs, but since https://github.com/JoshuaKGoldberg/create-typescript-app uses `lib`, let's use that here:

```suggestion
	outDir: "lib",
```

I'm soon going to automate keeping all my packages up-to-date with CTA and having discrepancies like this will make that more complicated.